sdk/search/search-documents/src/searchIndexClient.ts

```diff
@@ -145,7 +152,7 @@ export class SearchIndexClient {
     let apiVersion = this.apiVersion;
 
     if (options.apiVersion) {
-      if (!["2020-06-30-Preview", "2020-06-30"].includes(options.apiVersion)) {
+      if (!["2020-06-30", "2021-04-30-Preview"].includes(options.apiVersion)) {
```

Comment:
isn't this a breaking change? If the consumer passed in `"2020-06-30-Preview"` then after updating they'd get a runtime error

Reply:
Yes. This was discussed among all the SDKs and agreed upon since this is a preview version.